### ISSUE TITLE
Dominant Assurance Contracts

### ISSUE THREAD

**Issue by dominictarr** [dominictarr/feedopensource#7]
feed open source is an iterative assurance contract.
@joshmh informed of a variant, the Dominant Assurance Contract 
http://en.wikipedia.org/wiki/Assurance_contract#Dominant_Assurance_Contracts

It adds extra incentive to participate.
This could also be used to fund a salesperson/hustler/promoter to bring eyeballs to the project.

**Comment by NHQ** [comment]
Neat. I think I get it:  if I am the entrepreneur, I give somebody 1/10 btc to pledge 2 btc to the project.  If I get 10 bitcoins total pledged, that persons contributes the two btc, and you (the project keeper) pay me 1.5 btc for my work.  If I fail to get enough pledges, the others keep the 1/10 btc I gave them.    

I wonder if this works equally if the entrepreneur risks they own money, or if the project-people finance the incentive.

**Comment by dominictarr** [comment]
I'd expect that usually, the entrepreneur and the project is the same person.
Usually, there is certainly a hard problem that the project has to select that person,
because it's a very different set of skills to programming.

**Comment by bluntworks** [comment]
Bitcoin also allows for this though not sure if anyone has put it into practice https://en.bitcoin.it/wiki/Contracts#Example_3:_Assurance_contracts

**Comment by dominictarr** [comment]
yes, I plan to use that, but that will require adding support in a bitcoin wallet app, and unfortunately, none have support for that yet. At the scale of the prototype stage, the honor system is sufficient.
